client/package.json

```diff
@@ -23,6 +22,7 @@
     "testWatch": "react-scripts test --coverage --watchAll=true",
     "eject": "react-scripts eject"
   },
+  "proxy": "http://localhost:5000",
```

Comment:
Out of curiosity, what is this used for?

Reply:
Strange! If I'm understanding the documentation correctly (which I might not be), this is used to tell NPM to use an HTTP proxy for outgoing requests. I don't think we would want to do this by default, and I'm not sure how it would have gotten introduced to your configuration.

Reply:
I just removed it and it's working now.